xoto3/dynamodb/README.md

```diff
@@ -80,25 +80,30 @@ refetching, and eventually giving up to the utility.
 ```python
 import xoto3.dynamodb.write_versioned as wv
 
+user_table = wv.ItemTable('User')
+group_table = wv.ItemTable('Group')
+# ^ the above are convenience APIs and are pure utilities; they perform no
+# IO and contain no mutable state or clients.
+
 user_key = dict(id="bob")
 group_key = dict(pk="team42")
 
+
 def add_user_to_new_or_existing_group(t: wv.VersionedTransaction) -> wv.VersionedTransaction:
-    user = wv.require(t, "User", user_key)
+    user = user_table.require(user_key)(t)
```

Comment:
Even with the extra parenthesis looking at it in this PR I do feel is is cleaner and I like how verbose it is. IE saying that we are requiring this key on the users table.

Reply:
@jstephensxoi are you saying that you understand this to mean we require that a given key exists as part of the definition of the users table, or that we are requiring that a user with that key exists within the users table?

I just read some ambiguity into your last sentence, and want to make sure I understand what you're understanding. :)

Reply:
Yeah rereading my comment I see what you mean. How I am reading it is that in the user_table I require that a user with the id of bob must exist